skel/rust-api/src/filemanager.rs

```diff
@@ -0,0 +1,51 @@
+use axum::{extract::Query, Json};
+use serde::{Deserialize, Serialize};
+use utoipa::{IntoParams, ToSchema};
+
+use tracing::{ info };
+//type Store = Mutex<Vec<FileManager>>;
+// TODO: SQLx is the store backend, through db.rs
+
+/// Item to do.
+#[derive(Serialize, Deserialize, ToSchema, Clone)]
+pub struct FileManager {
```

Comment:
Is that the Rust equivalent of a Django Model class? 
Just wondering about the naming... the FileManager is more the microservice itself where the model would be more something like File/Object/FileObject/... ?

Not at all into Rust, so please ignore it that does not apply!

Reply:
Yes! I was torn between "File" and "Object", so I left it as "FileManager", which is clearly confusing and wrong, which one would you pick from the former two names? I'll change it :)

Reply:
No `File` doesn't clash with names, Rust is pretty good with shadowing names without clashes... let's go with file then.